QUARKUS-4221.md

```diff
@@ -85,7 +82,8 @@ the warning in the guide: [simple-command-line-application](https://quarkus.io/g
    New tests will be created in a new module quarkus-picocli.
 
 ### Impact on resources:
-  After the new test classes introduced in quarkus-picocli on JVM will be increased 6.381 seconds.
+  After the new test classes introduced in quarkus-picocli on JVM  baremetal will be increased 6.381 seconds,
```

Comment:
My English is generally not great, but I have doubt `on JVM baremetal` is how we usually say it. usually we say JVM mode / native mode and then on baremetal / OpenShift. Anyway, that's just opinion, treat it as you will.